srcpkgs/cgminer/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'cgminer'
 pkgname=cgminer
-version=4.10.0
+version=4.11.1
 revision=1
 build_style=gnu-configure
 configure_args="
```

Comment:
Why is the next line not on this one (or at least some of the configure options)?

Reply:
Accepted pattern for long lines.